packages/altair-koa-middleware/package.json

```
@@ -15,15 +15,15 @@
     "@types/supertest": "^2.0.11",
     "jest": "29.4.1",
     "koa": "^2.13.1",
-    "koa-send": "^5.0.1",
     "nodemon": "^2.0.12",
     "supertest": "^6.1.6",
     "ts-jest": "29.0.5",
     "ts-node": "^10.2.1",
     "typescript": "4.9.5"
   },
   "peerDependencies": {
-    "@koa/router": "^12.0.1"
+    "@koa/router": "^12.0.1",
```

Is this the first changes intended for the patch release, before making the final desired changes in a new major release? If so, to be as discussed in https://github.com/altair-graphql/altair/pull/2342#discussion_r1416774932 this should be back to the original peer dependency version requirement, which was `~11.0.2` not `^12.0.1`:

https://unpkg.com/altair-koa-middleware@5.2.12/package.json

An alternative approach is to do this, which could avoid a second major release:

```suggestion
    "@koa/router": "11.0.2 - 12",
```

That will support everything from v11.0.2 up to v12, and anything in the v12.x.x version range.

(assuming it wasn't important for you not to support v12.0.0, since you used `^12.0.1`).

If you didn't really care about skipping v11 smaller than v11.0.2, this works:

```suggestion
    "@koa/router": "11 - 12",
```